demo/meeting/src/types/index.ts

```diff
@@ -34,3 +34,17 @@ export type ContentShareControlContextType = {
   toggleContentShare: () => Promise<void>;
   togglePauseContentShare: () => void;
 };
+
+export enum DeviceLabels {
+  None,
+  Audio,
+  Video,
+  AudioAndVideo,
+};
+
+export enum MeetingMode {
+  Spectator,
+  Attendee,
+};
+
+export type DeviceLabelTrigger = () => Promise<MediaStream>;
```

Comment:
Is this used?

Reply:
It's not used now. But it could be used if we want to test the input `deviceLabels` of `DeviceLabelTrigger` type.

Deleted it in meeting demo and exported the one in the library. Thanks!